packages/ToggleSwitch/__tests__/ToggleSwitch.spec.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import { shallow, mount } from 'enzyme'
 
 import ToggleSwitch from '../ToggleSwitch'
 
```

Comment:
Could you please change `shallow` to `mount` where in snapshot test lines 9 and 12. This is needed for styles, generated by styled-components, to be printed in snapshot.